source/hdev/Applications/Advanced/ObjectMatching/SurfaceMatchingFindModel.hdev

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<hdevelop file_version="1.2" halcon_version="19.11.0.0">
+<hdevelop file_version="1.2" halcon_version="20.05.0.1">
```

Comment:
This happens when you save a hdev file using a specific (in this case 20.05) version of Halcon. This should be done for every file in the 'Update Halcon tested version to 20.05' commit I think.